src/main/java/com/qunite/api/web/controller/AuthenticationController.java

```diff
@@ -31,8 +33,11 @@
 @RestController
 public class AuthenticationController {
   private final UserService userService;
-  private final AuthResponseMapper responseMapper;
   private final UserMapper userMapper;
+  private final AuthResponseMapper authResponseMapper;
+
+  @Value("${jwt.access-token-expiration-time}")
+  private Integer expirationTime;
```

Comment:
As I have [written before](https://github.com/q-unite/qunite-api/pull/115#discussion_r1295072664), I do not meet this logic in controller. I think you should extend TokenPair with int expires field and use setter in JwtService.createJwt method